run_functional_tests.sh

```diff
@@ -32,6 +32,8 @@ echo done.
 # Finally run protractor
 echo Running tests...
 node_modules/protractor/bin/protractor protractor.config.js
+## Run functional tests on the bases of suite genere!  ##
+#node_modules/protractor/bin/protractor protractor.config.js --suite smokeTest //This line has been commented for now! 
```

Comment:
I think that what makes the most sense here is to pass the test suite name to the run_functional_tests.sh as a parameter:

`
node_modules/protractor/bin/protractor protractor.config.js --suite $1`